packages/account-v2/src/pages/LoginHistory/LoginHistory.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Table } from '@deriv-com/ui';
+
+type TRowData = {
+    action?: string;
+    browser?: string;
+    datetime?: string;
+    ipAddress?: string;
+    status?: string;
+};
+
+const headers = ['Date and Time', 'Action', 'Browser', 'IP Address', 'Status'] as const;
```

Comment:
```
const columnOrder = ['datetime', 'action', 'browser', 'ipAddress', 'status'];
const headers = {
  'datetime': 'Date and Time',
  ...
}
const columns = columnOrder.map(key => ({ header: headers[key] });
```

Reply:
Since your headers and data are not in order, you can create an array of keys to define the order.